.gitignore

```diff
@@ -30,4 +30,7 @@ binaries/
 .idea
 
 # Makefile plugin
-/.vscode/*.log
\ No newline at end of file
+/.vscode/*.log
+
+/Firefly-Kernel.code-workspace
```

Comment:
why ignore this when we alrdy have it in the repo, just let it be there?

Reply:
As long as nobody has any trouble with it, it's fine by me ig.